src/theme/components/pagination.ts

```diff
@@ -9,11 +9,6 @@ const Pagination = {
       lineHeight: "1.15",
       textDecoration: "none",
     },
-    li: {
-      _first: {
-        marginTop: "xxs",
-      },
-    },
   },
 };
```

Comment:
Also remove `marginBottom` from `baseStyle`.  Outside of the typography components, this is currently the only component that has built in spacing like this.

@EdwinGuzman Do you agree with making this change to bring `Pagination` in alignment with the other components?